sdk/tracing/azotel/otel.go

```diff
@@ -14,13 +14,16 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/tracing"
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/propagation"
 	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
 	"go.opentelemetry.io/otel/trace"
 )
 
 // TracingProviderOptions contains the optional values for NewTracingProvider.
 type TracingProviderOptions struct {
-	// for future expansion
+	// Propagator is the OpenTelemetry propagator to use.
+	// If not set, a propagation.TraceContext propagator is used.
+	Propagator propagation.TextMapPropagator
```

Comment:
Per our conversation, it sounds like this will be removed?

Reply:
Yes, I have removed it from the options